core/netparams/keys.go

```diff
@@ -34,7 +34,6 @@ const (
 	MarketSuccessorLaunchWindow              = "market.liquidity.successorLaunchWindowLength"
 
 	MarketLiquidityProvisionShapesMaxSize            = "market.liquidityProvision.shapes.maxSize"
-	MarketLiquidityTargetStakeTriggeringRatio        = "market.liquidity.targetstake.triggering.ratio"
```

Comment:
Instead of removing it entirely, which counts as a breaking change and means we'll have to go around and remove the nework parameter from all the system tests, and all the example configs in the vegacapsule repo etc etc, can you instead just add the parameter to the deprecated list?
```
var Deprecated = map[string]struct{}{
	StakingAndDelegationRewardPayoutFraction:    {},
	StakingAndDelegationRewardPayoutDelay:       {},
	StakingAndDelegationRewardMaxPayoutPerEpoch: {},
}
```

this will mean that core won't add it to any newly generated genesis files, you can't propose to change the value, and then we can fully remove it peacefully after the release?

Reply:
Thank you, I'll look into doing that.